apis/apps/v1alpha1/nebulacluster_metad.go

```diff
@@ -240,11 +241,22 @@ func (c *metadComponent) GenerateVolumeMounts() []corev1.VolumeMount {
 			SubPath:   "data",
 		},
 	}
+
+	if licensePath, ok := c.nc.Spec.Metad.Config["license_path"]; ok {
```

Comment:
`licensePath` maybe `""` ?

Reply:
It is not easy for users to copy license files to pods?

How about let user specifies `Secert` name and key instead of using `license_path`?